### PR TITLE
Feature controls rework

### PR DESCRIPTION
**Pull request by Sammyboyo**
Changing controls to use any state inputs and output to any number of motors.

### CODE REVIEW

File: src/controls/estimator_manager.hpp [hunk 1]
```diff
@@ -19,7 +19,6 @@
 
 #define NUM_IMU_CALIBRATION 50000
 
-
 // Rev encoder pins
```
Comment (Pandabear1125) [inline src/controls/estimator_manager.hpp:22]: Should this be moved into the rev encoder's driver files?

File: teensy4/HardwareSerial1.cpp
Comment (Pandabear1125) [inline teensy4/HardwareSerial1.cpp:1]: We should revert the Serial buffer size increases

**Review by Pandabear1125 — CHANGES_REQUESTED** [review]

File: src/controls/estimator_manager.hpp (hunk 1 above)
Comment (Rishi-H) [reply src/controls/estimator_manager.hpp:22]: nah, I don't think so, since we init the encoders in estimator manager.

**Review by Pandabear1125 — APPROVED** [review]
appears good

**Pull request merged**